internal/factory/gate.go

```
@@ -0,0 +1,302 @@
+	if err != nil {
+		return "", fmt.Errorf("inspect repository Git metadata: %w", err)
+	}
+	if !sourceInfo.IsDir() {
+		return "", errors.New("repository Git metadata must be a directory")
+	}
+
+	projectionParent := filepath.Join(filepath.Dir(worktreePath), ".factory-git")
+	projection := filepath.Join(projectionParent, runID)
+	if projectionInfo, statErr := os.Stat(projection); statErr == nil {
+		if !projectionInfo.IsDir() {
+			return "", fmt.Errorf("Git metadata projection %q is not a directory", projection)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)